tutorials/scripting/c_sharp/c_sharp_style_guide.rst

```diff
@@ -262,7 +262,7 @@ Rather than:
 
 .. code-block:: csharp
 
-    FindNode()?.Change(wpnDmg);
+    GetNode()?.Change(wpnDmg);
```

Comment:
This isn't necessarily correct, this isn't the same method as it has no arguments provided

Reply:
Correct